src/ansys/fluent/core/fluent_connection.py

```diff
@@ -520,6 +517,12 @@ def __init__(
         )
         FluentConnection._monitor_thread.cbs.append(self._finalizer)
 
+    @property
+    @deprecated(version="0.32.dev0", reason="No longer required at this level.")
```

Comment:
Should the versions shown in the deprecation table always be some release version?

Reply:
Agree 👍